docs/source/support/troubleshooting.rst

```diff
@@ -0,0 +1,84 @@
+===============
+Troubleshooting
+===============
```

Comment:
I have a feeling that this info should be displayed by CLI when the corresponding error is encountered, instead of exposing some `nucypher` internals. But that is perhaps low priority.

Reply:
There are definitely areas that can be cleaned up in the CLI. You can file a separate issue.

Reply:
@fjarri I opened #1638 to cover at least the ones related with transactions.

Reply:
Cool, thank you.

Reply:
I filed a more general issue #1642 about it.